cjs/interface/event-target.js

```diff
@@ -6,4 +6,33 @@ const EventTarget = (m => /* c8 ignore start */ m.__esModule ? m.default : m /*
 /**
  * @implements globalThis.EventTarget
  */
-exports.EventTarget = EventTarget;
+class DOMEventTarget extends EventTarget {
+
+  /**
+   * @protected
+   */
+  _getParent() {
+    return null;
+  }
+
+  dispatchEvent(event) {
+    const dispatched = super.dispatchEvent(event);
+
+    // intentionally simplified, specs imply way more code: https://dom.spec.whatwg.org/#event-path
+    if (dispatched && event.bubbles && !event.cancelBubble) {
+      const parent = this._getParent();
+      if (parent && parent.dispatchEvent) {
+        const options = {
+          bubbles: event.bubbles,
+          cancelable: event.cancelable,
+          composed: event.composed,
+        };
+        // in Node 16.5 the same event can't be used for another dispatch
+        return parent.dispatchEvent(new event.constructor(event.type, options));
```

Comment:
Changed to cope with Node 16.5 restrictions. I don't know if there is a better way to do that

Reply:
FWIWI the same happens in the DOM.

Reply:
Interesting, thanks!